packages/osd-plugin-generator/package.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "@osd/cross-platform": "1.0.0",
     "@osd/dev-utils": "1.0.0",
-    "ejs": "^3.1.7",
+    "ejs": "^3.1.10",
```

Comment:
I'm assuming the title should be update to 3.1.10 instead of 3.1.101?

Reply:
+1

Reply:
Yes. `1` should be `. Will correct it and thank you for the careful check.